diff --git a/gni/v8.gni b/gni/v8.gni
index 09c3e7adec..fda3fb2c7a 100644
--- a/gni/v8.gni
+++ b/gni/v8.gni
@@ -148,7 +148,7 @@ declare_args() {
   cppgc_enable_2gb_cage = false
 
   # Enable support for larger cages, up to 16GB.
-  cppgc_enable_larger_cage = true
+  cppgc_enable_larger_cage = false
 
   # Enable advanced BigInt algorithms, costing about 10-30 KB binary size
   # depending on platform. Disabled on Android to save binary size.
